### PR TITLE
Gen command

### PR DESCRIPTION
**Pull request by stevelr**
Add 'gen' command - fixes wash #163 
Update dependencies: upgraded oci_distribution from 0.6.0 to 0.7.0, cargo from 0.55 to 0.56
Bumped wash crate to 0.6.1

### CODE REVIEW

File: src/smithy.rs [hunk 1]
```diff
@@ -185,3 +231,65 @@ fn select_config(opt_config: &Option<PathBuf>) -> Result<CodegenConfig, anyhow::
...
+    let mut config = select_config(&opt.config)?;
+    if !opt.lang.is_empty() {
+        config.output_languages = opt.lang.clone()
+    }
+    let mut input_models = Vec::new();
+    std::mem::swap(&mut config.models, &mut input_models);
+    let model = build_model(opt.input, input_models, config.base_dir.clone(), verbose)?;
+
+    let templates = if let Some(ref tdir) = opt.template_dir {
+        println!("importing templates from {}", &tdir.display());
```
Comment (autodidaddict) [inline src/smithy.rs]: this looks like a debug statement. Does this still belong in the final code?

File: src/smithy.rs [hunk 2]
```diff
@@ -27,6 +28,14 @@ pub(crate) struct ValidateCli {
     opt: ValidateOptions,
 }
 
+/// Generate using codegen.toml
```
Comment (autodidaddict) [inline src/smithy.rs]: Generate what? I think this help line (which shows up when you just type `wash` with no parameters) should indicate at least the category of things that can be generated here.

**Review by autodidaddict — DISMISSED** [review]
Just a recommendation on a doc change and removing what looks like a debug output line

File: src/smithy.rs (hunk 1 above)
Comment (stevelr) [reply src/smithy.rs]: it's useful output, but I changed it to improve the formatting

File: src/smithy.rs (hunk 2 above)
Comment (stevelr) [reply src/smithy.rs]: updated to "Generate code from smithy IDL files"

**Review by brooksmtownsend — APPROVED** [review]
LGTM

**Pull request merged**